oat-sa/extension-tao-testqti
# Feature/tca 424/extended time using adjustments

During passing test via LTI with custom parameter  'extended_time=2' the timer shows incorrect time for second section, if for first section timer is over
 
Related to : https://oat-sa.atlassian.net/browse/TCA-424
 
`extraTime` parameter should not be changed on the `get` action.

#### Steps to reproduce  (for bugs)
 - Open http://ltiapps.net/test/tc.php, fill all required fields
- Tick checkbox 'Display optional parameters?' in LTI Consumer emulator.
 - Set extended_time=2 in 'Custom parameters' in 'Message data' block.
 - Click 'Save data', 'Launch in new window'.
 - Launch the test and wait until time is over in first section
  
#### How to test
 
 - run tests https://github.com/oat-sa/extension-tao-testqti/tree/cdc43c3d529c3b4e2378b50c604e3135c2786fcb/test/integration/runner/time
or manually:
 - check scenario from reproduce section with a fix
 
#### Dependencies

Companion PR :
 - [ ] https://github.com/oat-sa/extension-tao-proctoring/pull/805
 - [ ] https://github.com/oat-sa/extension-lti-proctoring/pull/136
